packages/odyssey-react-mui/src/MenuButton.tsx

```diff
@@ -65,10 +82,25 @@ export type MenuButtonProps = {
    * If the MenuButton is an overflow menu or standard menu.
    */
   isOverflow?: boolean;
+  /**
+   * The horizontal alignment of the menu.
+   */
+  menuAlignment?: "left" | "right";
+  /**
+   * Optional function to fire on button click
+   */
+  onClick?: ButtonProps["onClick"];
+  /**
+   * Optional function to fire on memu close
+   */
+  menuPaperRef?: MutableRefObject<HTMLDivElement>;
```

Comment:
Why do we need this ref?

Reply:
If we need a ref, maybe we can make an internal version of this component and expose only the one without this ref?